src/test/java/code401challenges/InsertionSortTest.java

```diff
@@ -0,0 +1,20 @@
+package code401challenges;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class InsertionSortTest {
+    @Before
+    public void setup() {
+
+    }
+    @Test
+    public void insertTest(){
+        int[] testarr = {3,5,2,4,1,7,6};
+        int[] answer = {1,2,3,4,5,6,7};
+        assertArrayEquals(answer,
+        InsertionSort.insertSort(testarr));
+    }

```

Comment:
This is a good test for "normal" case, but you definitely want to test edge cases, : array length 0, array length 1, array with duplicate numbers, array with negative numbers, etc.